backend/app/usecase/feature/dynamic_test.go

```diff
@@ -46,11 +52,96 @@ func TestDynamicDecisionMaker_IsFeatureEnable(t *testing.T) {
 				"example-feature": {
 					ID:        "example-feature",
 					IsEnabled: true,
+					Type:      entity.ManualToggle,
 				},
 			},
 			featureID:         "example-feature",
 			expectedIsEnabled: true,
 		},
+		{
+			name: "permission toggle, feature enabled, permission checker not defined",
+			toggles: map[string]entity.Toggle{
+				"example-feature": {
+					ID:        "example-feature",
+					IsEnabled: true,
+					Type:      entity.PermissionToggle,
+				},
+			},
+			featureID: "example-feature",
+			roles: map[string][]role.Role{
+				"id": {role.Basic},
```

Comment:
```suggestion
				"id": {role.Admin},
```
We need to make sure that the feature is not shown only because of the toggle is not enabled.